lxl-web/src/routes/(app)/[[lang=lang]]/find/search.ts

```diff
@@ -154,13 +166,14 @@ function displayMappings(
 function displayFacetGroups(
 	view: PartialCollectionView,
 	displayUtil: DisplayUtil,
-	locale: LangCode
+	locale: LangCode,
+	translate: translateFn
 ): FacetGroup[] {
 	const slices = view.stats?.sliceByDimension || {};
 
 	return Object.values(slices).map((g) => {
 		return {
-			label: g.dimension, // TODO
+			label: translate(`facet.${g.dimension}`),
```

Comment:
Yep, however the interface only blows up in dev mode when no translations found:

https://github.com/libris/lxlviewer/blob/0cf990205a1b792c87f50ca7ac2837b8a127244c/lxl-web/src/lib/i18n/index.ts#L40-L43

Otherwise it just returns the key. We can disable this if it's too annoying...

Reply:
Ok!